src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPage.tsx

```diff
@@ -74,18 +73,18 @@ function QuickbooksCompanyCardExpenseAccountPage({policy}: WithPolicyConnections
                                 onToggle={(isOn) => Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.AUTO_CREATE_VENDOR, isOn)}
```

Comment:
The `autoCreateVendor` of the `config` field is used to store following configuration instead of the default vendor.

When the user toggles the default vendor switch, I believe we need to assign `NONE` to `nonReimbursableBillDefaultVendor`. (Please see the second screenshot attached to this message)

<img width="400" alt="Screenshot 2024-05-06 at 9 42 23 AM" src="https://github.com/Expensify/App/assets/98560306/f3bc1fae-7bc2-4fe0-b418-26b729b580cf">

<img width="400" alt="Screenshot 2024-05-06 at 9 55 50 AM" src="https://github.com/Expensify/App/assets/98560306/333ea330-a2f5-475e-b3af-8ef178504bc6">

Reply:
Fixed